src/split-descriptors.js

```diff
@@ -10,7 +10,7 @@
 // Each element of the resulting array is a subarray of the original Uint8Array.
 export default function splitDescriptors(bytes) {
     const descs = [];
-    if (Object.prototype.toString.call(bytes) !== '[object Uint8Array]') {
+    if (bytes instanceof Uint8Array) {
```

Comment:
Actually, this should be `!(bytes instanceof Uint8Array)`!

Reply:
This must be Friday :)